packages/storybook/stories/ErrorBar/ErrorBar.stories.tsx

```diff
@@ -25,6 +25,10 @@ const getDocs: () => JSX.Element = () => {
         `ErrorBar` is a wrapper on fluent UI's `MessageBar` with additional features for surfacing Azure Communication
         Services errors on the UI consistently.
       </Description>
+      <Description>
+        Set the `showErrorBar` option for `ChatComposite` to use an `ErrorBar` to show errors. This feature is still
```

Comment:
I would recommend removing the `This feature...` line from here.
We are still in Beta and this is a given anyway.
Plus we might forget to remove this piece of text once ErrorBar becomes stable.